Sources/CodegenCLI/Commands/Initialize.swift

```diff
@@ -154,16 +217,25 @@ extension ApolloCodegenConfiguration {
         "schemaTypes" : {
           "path" : "./\(schemaName)",
           "moduleType" : {
-            \(supportCocoaPods ? "\"other\"" : "\"swiftPackageManager\"") : {
-            }
+            "\(moduleType)" : {
+            \(moduleTarget)
```

Comment:
I could be wrong, but this looks like the indentation is off, and that it may be missing a `}`? Not sure. Do we have unit tests for this? (I have another comment in the `InitializeTests` asking about testing this as well.)

Reply:
The JSON is correct, otherwise none of the decoding from the return string of `minimalJSON` into an `ApolloCodegenConfiguration` instance would succeed.